packages/control-input/src/ControlGroup.tsx

```diff
@@ -37,44 +38,49 @@ export const ControlGroup = ({
 }: ControlGroupProps) => {
 	const { groupId, hintId, messageId } = useControlGroupIds(id);
 	const describedByIds = [
-		hint ? hintId : null,
 		message ? messageId : null,
+		hint ? hintId : null,
 	].filter(Boolean);
-	const describedBy = describedByIds.length
+	const describedByIdString = describedByIds.length
 		? describedByIds.join(' ')
 		: undefined;
 	return (
-		<FieldContainer invalid={invalid} id={groupId}>
-			<fieldset
-				aria-describedby={describedBy}
-				css={{ padding: 0, margin: 0, border: 'none' }}
-			>
-				{label ? (
-					<FieldLabel as="legend" required={required}>
-						{label}
-					</FieldLabel>
-				) : null}
-				<Stack
-					gap={0.5}
-					css={{ marginTop: label ? mapSpacing(0.5) : undefined }}
+		<ControlGroupContext.Provider
+			value={{ groupMessageId: describedByIdString }}
```

Comment:
```suggestion
			value={{ describedBy }}
```

Maybe this for consistency?